README.md

```diff
@@ -29,7 +29,7 @@
 
 We've come a long way, but this project is still in Alpha, lots of development is happening, API might change, beware of the Dragons 🐉..
 
-**Want to get started?** Check our [examples folder](/examples). You can check the development status at the [Waffle Board](https://waffle.io/libp2p/js-libp2p).
+**Want to get started?** Check our [examples folder](/examples).
 
 [![Throughput Graph](https://graphs.waffle.io/libp2p/js-libp2p/throughput.svg)](https://waffle.io/libp2p/js-libp2p/metrics/throughput)
```

Comment:
We should remove the throughput graph below as that is also referencing waffle.
```suggestion
```

Reply:
Sure, should have grabbed README :D